### PR TITLE
Resolution for Issue #416

### PR DESCRIPTION
**Pull request by Colin-Holehouse**
The included code updates are for issue #416

### CODE REVIEW

File: src/pdngen/test/issue_416.ok [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Notice 0: Reading LEF file:  nangate45/NangateOpenCellLibrary.mod.lef
```
Comment (jjcherry56) [inline src/pdngen/test/issue_416.ok:1]: not convinced "issue_416" is very descriptive; the issues may not have much of a lifetime when we start reorganizing the repos

Comment (Colin-Holehouse) [reply src/pdngen/test/issue_416.ok]: Updated the test to be extend_to_core_ring, which is what the test is checking.

**Comment by jjcherry56** [comment]
i would approve but don't appear to be a requested reviewer so i can't

**Comment by rovinski** [comment]
@jjcherry56 you can self-request a review

**Pull request merged**